src/CSET/operators/regrid.py

```diff
@@ -174,9 +175,10 @@ def regrid_onto_xyspacing(
     latout = np.arange(lat_min, lat_max, yspacing)
     lonout = np.arange(lon_min, lon_max, xspacing)
 
-    if callable(getattr(iris.analysis, method)):
+    regrid_method = getattr(iris.analysis, method, None)
```

Comment:
`regrid_method = getattr(iris.analysis, method, None)` -> `    regrid_method = getattr(iris.analysis, method, None)`

Reply:
I don't think it needs indenting? I saved the regrid method to a variable so we only need to get it once, and then it is checked in the if statement.